extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3409,6 +3409,17 @@ static public function cityIDtoDomain( $city_id, $master = false ) {
 		return $url;
 	}
 
+	/**
+	 * Returns local address of href
+	 *
+	 * @param int $city_id
+	 * @param string $href
+	 * @return string
+	 */
+	static public function cityIdToLanguagePath( $city_id, $href ) {
+		return static::cityUrlToLanguagePath( WikiFactory::cityIDtoUrl( $city_id ) ) . $href;
```

Comment:
I'd use `static::cityIDtoUrl` here rather than spelling out the class, and I'd suggest removing the `$href` parameter so that this method is usable for other use cases and matches similar methods like `cityIDtoUrl` and just returns the language path.